### PR TITLE
[SYCLomatic] Support the migration of some 64-bit cublas API and refine the migration of related 32-bit API 

### PR DESCRIPTION
**Pull request by zhiweij1**
`cublas<T>nrm2_v2_64`
`cublas<T>dot_v2_64`
`cublas<T>dotu_v2_64`
`cublas<T>dotc_v2_64`
`cublas<T>scal_v2_64`
`cublas<T>axpy_v2_64`
`cublas<T>copy_v2_64`
`cublas<T>swap_v2_64`
`cublas<T>asum_v2_64`

Signed-off-by: Jiang, Zhiwei <zhiwei.jiang@intel.com>

### CODE REVIEW

**Review by tomflinda — APPROVED** [review]
LGTM

File: clang/runtime/dpct-rt/include/dpct/blas_utils.hpp [hunk 1]
```diff
@@ -207,6 +207,12 @@ using wrapper_int64_out =
     parameter_wrapper_t<std::int64_t, std::int64_t, parameter_inout_prop::out>;
```
Comment (zhiweij1) [inline clang/runtime/dpct-rt/include/dpct/blas_utils.hpp:207]: Can the wait in line 194 be removed?

File: clang/runtime/dpct-rt/include/dpct/blas_utils.hpp [hunk 2]
```diff
@@ -207,6 +207,12 @@ using wrapper_int64_out =
...
 using wrapper_float_out =
```
Comment (zhiweij1) [inline clang/runtime/dpct-rt/include/dpct/blas_utils.hpp:208]: can the copyback event be removed?

File: clang/runtime/dpct-rt/include/dpct/blas_utils.hpp [hunk 3]
```diff
@@ -207,6 +207,12 @@ using wrapper_int64_out =
...
     parameter_wrapper_t<float, float, parameter_inout_prop::out>;
```
Comment (zhiweij1) [inline clang/runtime/dpct-rt/include/dpct/blas_utils.hpp:209]: get_buffer can use get_buffer_and_offset instead.

**Review by ziranzha — APPROVED** [review]

File: clang/runtime/dpct-rt/include/dpct/blas_utils.hpp (hunk 1 above)
Comment (zhiweij1) [reply clang/runtime/dpct-rt/include/dpct/blas_utils.hpp:207]: I will fix it.

File: clang/runtime/dpct-rt/include/dpct/blas_utils.hpp (hunk 2 above)
Comment (zhiweij1) [reply clang/runtime/dpct-rt/include/dpct/blas_utils.hpp:208]: Yes, I will fix it.

**Pull request merged**